File: testdata/cpp/level-01-basic-decls/src/main.cpp
#include <string>

const int MAX_RETRIES = 3;

bool debug = false;

std::string hello() {
    return "hello";
}

int add(int a, int b) {
    return a + b;
}

int main() {
    hello();
    int result = add(1, 2);
    return 0;
}

File: testdata/go/level-01-basic-decls/src/main.go
package main

import "fmt"

const MaxRetries = 3

var Debug bool

func Hello() string {
	return "hello"
}

func add(a, b int) int {
	return a + b
}

func main() {
	fmt.Println(Hello())
	result := add(1, 2)
	_ = result
}
